.github/workflows/build-centos-pypa-rust.yml

```diff
@@ -55,14 +61,20 @@ jobs:
           context: ./centos-pypa-rust-aarch64
           file: ./centos-pypa-rust-aarch64/Dockerfile
           push: true
-          tags: ${{ steps.dockertag.outputs.dockertag }}
+          tags: ${{ steps.meta.outputs.tags }}
 
-      - name: Set docker tag to build
+      - name: Docker meta
         if: matrix.arch == 'x86_64'
-        id: dockertag1
-        run: |
-          # shellcheck disable=SC2296,SC2086
-          echo 'dockertag=ghcr.io/chia-network/centos-pypa-${{ matrix.arch }}:${{ github.sha }}' >>$GITHUB_OUTPUT
+        id: meta
```

Comment:
The step ID "meta" is used twice in the same job, so GitHub Actions runs on this workflow are failing, see https://github.com/Chia-Network/build-images/actions/runs/4613035978

Reply:
Fixed